docs/reference/cloud/list-of-supported-clouds/the-google-gce-cloud-and-juju.md

```diff
@@ -96,3 +96,11 @@ Base path to look for machine disk images.
 | {ref}`placement-directive-system-id`             | &#10005; |
 | {ref}`placement-directive-zone`                  | &#10003; |
 
+## Cloud-specific storage providers
+
+> See first: {ref}`storage-provider`
+
+(storage-provider-gce)=
+### `gce`
+
+The GCE provider does not currently have any specific configuration options.
```

Comment:
It is supposed to support volume type but there's a bug
https://github.com/juju/juju/issues/20349

Reply:
Added:

```
Configurations options:

- `type`. Value is `pd-ssd`. Warning: [bug](https://github.com/juju/juju/issues/20349).
```

Hope I got it right!